packages/fhir/test/Adaptor.test.js

```diff
@@ -183,3 +185,22 @@ describe('getClaim', () => {
     expect(finalState.data).to.eql(fixtures.claim);
   });
 });
+
+describe('extractResource', () => {
+  const baseUrl = 'https://hapi.fhir.org/baseR4';
+
+  it('should extract resources of the specified type from the bundle', async () => {
+    const bundleResponse = await axios.get(`${baseUrl}/Bundle`);
```

Comment:
For `bundleResponse` use client fixtures, See example here - [packages/fhir/test/ClientFixtures.js](https://github.com/OpenFn/adaptors/blob/main/packages/fhir/test/ClientFixtures.js)